api/organization_members.go

```diff
@@ -16,22 +16,22 @@ import (
 	"go.vocdoni.io/dvote/log"
 )
 
-// organizationMembersHandler godoc
+// organizationUsersHandler godoc
```

Comment:
Shouldn;t this file be now become `organization_users.go`? Thesame for the test file